### ISSUE TITLE
Dark Mode

### ISSUE THREAD

**Issue by ebelinski** [ProtonMail/ios-mail#2]
It would be awesome to see the ProtonMail iOS app adopt [Dark Mode](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/dark-mode/). With more and more apps adopting it, users will increasingly expect that actively maintained apps will conform to their Light/Dark appearance system setting.

Relevant resources:

* [Supporting Dark Mode in Your Interface](https://developer.apple.com/documentation/xcode/supporting_dark_mode_in_your_interface)
* [WWDC 214 Implementing Dark Mode on iOS](https://developer.apple.com/videos/play/wwdc2019/214/)
* [Dark Mode – iOS Human Interface Guidelines](https://developer.apple.com/design/human-interface-guidelines/ios/visual-design/dark-mode/)

**Comment by abjurato** [comment]
Thank you! We are working on Dark Mode, Dynamic Type and better VoiceOver support as part of Version 4.0 redesign

**Comment by blcarson** [comment]
@abjurato Is there a way to signup for beta? I'd love to test that out when its ready.

**Comment by bojanstef** [comment]
How can I help @abjurato

**Comment by derekmorr** [comment]
Any idea when this will be released?

**Comment by BrianEnigma** [comment]
Is there a branch for the v4 UI redesign? Unless I'm mistaken, I only see `master`. Are there design docs? In the interim, would it be possible to just change the foreground/background color constants from their legacy values to the new constants for Default Label Color / System Background Color, which automatically adapt to light and dark mode?

I'd love to have the other accessibility bells and whistles like system-dynamic font sizes, but even the lack of light/dark mode would be a good step forward in terms of accessibility.

**Comment by derekmorr** [comment]
I don’t believe there’s a branch for it. It looks like Proton does their development in a private repo and pushes to this public one periodically (when they have a release). I’m not aware of any design docs on the new UI.

It’s quite frustrating. This issue has been open for over a year.

**Comment by mattgeowild** [comment]
Any updates? @abjurato

**Comment by cantonalex** [comment]
Any updates?

**Comment by BrianEnigma** [comment]
It appears that the latest beta (3.0.2 6250) has dark mode, but not the other accessibility features like dynamic font size.
